storage/persistence/spanner/setup_test.go

```diff
@@ -83,30 +67,27 @@ func CreateDatabase(ctx context.Context, uri string) error {
 		return nil
 	}
 
+	ddlBytes, err := ioutil.ReadFile("../../../scripts/schemas/spanner/ddls.sql")
+	if err != nil {
+		return fmt.Errorf("read ddls file: %w", err)
+	}
+	ddls := make([]string, 0)
+	for _, ddl := range strings.Split(string(ddlBytes), ";") {
```

Comment:
There is only one table to be created, line 75 seems to be ineffective.

Reply:
Not actually, at least we need to add some indexes like `pool_name + ready_time` to speedup the query. Besides, I think it's ok to ineffective in test case, it harms nothing. How do you think?

Reply:
Yes, I agree.